tests/utils/dev-server.js

```diff
@@ -35,7 +35,7 @@ const startServer = async ({ cwd, env = {}, args = [] }) => {
           close: async () => {
             selfKilled = true
             const pids = await pidtree(ps.pid).catch(() => [])
-            pids.forEach((pid) => () => {
+            pids.forEach((pid) => {
```

Comment:
I believe this was a bug. What do you think @erezrokah ?

Reply:
Yes I think so. A bad shameful copy paste probably